frontend/ui/game.tsx

```diff
@@ -38,16 +41,47 @@ export class Game extends React.Component {
     }
   }
 
-  public componentWillMount() {
+  public componentDidMount(prevProps, prevState) {
     window.addEventListener('keydown', this.handleKeyDown.bind(this));
+    this.setDarkMode(prevProps, prevState);
+    this.setTurnIndicatorFavicon(prevProps, prevState);
     this.refresh();
   }
 
   public componentWillUnmount() {
     window.removeEventListener('keydown', this.handleKeyDown.bind(this));
+    document.getElementById("favicon").setAttribute("href", defaultFavicon);
```

Comment:
Oh yea I wasn't sure how to test this codepath. I see you do some logic in `componentWillUnmount`, but afaik this component never unmounts? If you click to go back to the homepage the whole app refreshes